salt/modules/rsync.py

```diff
@@ -68,7 +68,8 @@ def rsync(src,
           exclude=None,
           excludefrom=None,
           dryrun=False,
-          rsh=None):
+          rsh=None,
+          additional_opts=None):
```

Comment:
Documentation is always a good thing.  Adding in for additional_opts as well as the others.